File: exercicios/Python/1154/1154.py
x = 100
i = 0
soma =0
while x >= 0:
    n = int(input())
    if n < 0:
        break
    if n >= 0:
        i += 1
        soma = soma + n
media = soma / i
print('{:.2f}'.format(media))

File: exercicios/R/1154/1154.r
input = file('stdin','r')

soma = as.integer(0)

media = as.double(0)

i = as.double(0)

n = as.integer(readLines(input,n=1))

while(n >=0){
    soma = soma + n
    i = i + 1

    n = as.integer(readLines(input,n=1))
}

media = soma /i;
write(sprintf("%.2f",media),'')